os-rsyslog.te

```diff
@@ -0,0 +1,10 @@
+policy_module(os-rsyslog,0.1)
+
+gen_require(`
+        type container_file_t;
+        type syslogd_t;
+        class file { append create open read write };
+')
+
+# LP #1810422
+allow syslogd_t container_file_t:file { append create open read write };
```

Comment:
I think you should do
```
manage_files_pattern(syslogd_t, container_file_t)
manage_dirs_pattern(syslogd_t, container_file_t)
manage_lnk_files_pattern(syslogd_t, container_file_t)
```

Reply:
Not really that I know about. 

If  you look at existing policy you will see them all over the place though.

Reply:
Yeah, they're in policy/support in the SELinux tree:

https://github.com/fedora-selinux/selinux-policy/tree/rawhide/policy/support

Reply:
Not really "documented", but... I mean, it's self-documenting code, so ... ;)